src/sections/pools/pool/positions/LiquidityPosition.tsx

```diff
@@ -148,61 +171,74 @@ export const LiquidityPosition = ({
-        >
-          <div sx={{ flex: "column", gap: 6, pr: 12 }}>
+        <div css={{ display: "grid", gridTemplateColumns: "1fr 1fr 1fr" }}>
+          <div sx={{ flex: "column", gap: 6 }}>
             <Text fs={14} color="whiteish500">
-              {t("liquidity.asset.positions.position.amount")}
-            </Text>
-            <Text>
-              {t("liquidity.asset.positions.position.shares", {
-                shares: position.shares,
-                fixedPointScale: meta.data?.decimals.toString() ?? 12,
-              })}
+              {t("liquidity.asset.positions.position.initialValue")}
             </Text>
-          </div>
-          <Separator orientation="vertical" />
-          <div sx={{ flex: "column", align: "center", px: 12 }}>
-            <div sx={{ flex: "column", gap: 6 }}>
-              <Text fs={14} color="whiteish500">
-                {t("liquidity.asset.positions.position.initialValue")}
-              </Text>
+            <div>
               <Text>
                 {t("value.token", {
                   value: position.providedAmount,
                   fixedPointScale: meta.data?.decimals.toString() ?? 12,
                   numberSuffix: ` ${meta.data?.symbol ?? "N/A"}`,
                 })}
               </Text>
-            </div>
-          </div>
-          <Separator orientation="vertical" />
-          <div sx={{ flex: "column", gap: 6, align: "center", pl: 12 }}>
-            <div sx={{ flex: "column", gap: 6 }}>
-              <Text fs={14} color="whiteish500">
-                {t("liquidity.asset.positions.position.currentValue")}
-              </Text>
-              <div sx={{ flex: "column", align: "start" }}>
-                <WalletAssetsHydraPositionsData
-                  symbol={position.symbol}
-                  value={position.value}
-                  lrna={position.lrna}
-                />
+              {providedAmountPriceLoading ? (
+                <Skeleton width={50} height={7} />
+              ) : (
                 <DollarAssetValue
-                  value={position.valueDisplay}
+                  value={providedAmountPrice}
                   wrapper={(children) => (
                     <Text fs={[11, 12]} lh={[14, 16]} color="whiteish500">
                       {children}
                     </Text>
                   )}
                 >
-                  <DisplayValue value={position.valueDisplay} />
+                  <DisplayValue value={providedAmountPrice} />
                 </DollarAssetValue>
-              </div>
+              )}
+            </div>
+          </div>
+          <Separator orientation="vertical" />
+          <div sx={{ flex: "column", gap: 6 }}>
+            <div sx={{ display: "flex", gap: 6 }}>
+              <Text fs={14} color="whiteish500">
+                {t("liquidity.asset.positions.position.currentValue")}
+              </Text>
+              <InfoTooltip
+                text={
+                  infoTooltipLoading ? (
```

Comment:
The content of the tooltip is a little bit different from Figma. The value should be on the second row